sdk/nodejs/appmesh/virtualRouter.ts

```diff
@@ -28,7 +28,7 @@ import * as utilities from "../utilities";
  * import * as aws from "@pulumi/aws";
  *
  * const serviceb = new aws.appmesh.VirtualRouter("serviceb", {
- *     meshName: aws_appmesh_mesh_simple.id,
+ *     meshName: aws_appmesh_mesh.simple.id,
```

Comment:
I know this isn't related to this PR - but is this "right" in terms of codegen?  This is a reference to an non-existant variable - so there's sort of no real notion of "right" - but the previous code looks more like what I would expect for a reference to an undefined variable.

cc @pgavlin @lblackstone

Reply:
I believe that this was caused by the change from 0.11 to 0.12 syntax: https://github.com/terraform-providers/terraform-provider-aws/commit/d47e82f170c20e655a30557289ef3772c44e13a8#diff-0601a42442cec4be2db943fba3998311

With TF12, we render this into a traversal without trying to infer the kind of the missing reference. We could improve that and retain the original rendering by doing that inference in `tf2pulumi`.